### PR TITLE
feat(hubble-ui): Add probes for Hubble-ui

### PR DESCRIPTION
**Pull request by mkilchhofer**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [na ] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Are you a user of Cilium? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md)
- [x] Thanks for contributing!

As this is a common pattern for all Kubernetes pods, I think it doesn't require additional use cases.
But: I contributed the `/healthz` endpoint only a few days ago. The feature for the backend requires hubble-ui >=0.12:
- https://github.com/cilium/cilium/pull/27011
- https://github.com/cilium/hubble-ui/releases/tag/v0.12.0

Resolves: https://github.com/cilium/hubble-ui/issues/586

```release-note
Hubble-ui now supports liveness and readiness probes
```

/cc: @geakstr

### CODE REVIEW

**Review by gandro — COMMENTED** [review]
Looks good to me. 

Please note that you will have to update the `values.yaml.tmpl` file instead of the `values.yaml` file and run `make -C install/kubernetes` and `make -C Documentation` to update all generated files relating to Helm.

**Review by gandro — APPROVED** [review]
Helm-wise looks good to me! Thanks! Let's also get a review from a Hubble UI code owner

**Review by geakstr — APPROVED** [review]

**Comment by geakstr** [comment]
LGTM!

**Review by youngnick — APPROVED** [review]

**Comment by youngnick** [comment]
Hi @mkilchhofer, sorry, but with #27002 merging, there's been a change to the required jobs to pass CI, so I recommend rebasing and then one of us can re-trigger the testing.

**Comment by mkilchhofer** [comment]
@youngnick @gandro rebased as requested :-)

**Pull request merged**
